machine-learning/_posts/2018-01-15-translearning.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: review
+title:  "Unsupervised Transformation Learning
+via Convex Relaxations"
+tags:   unsupervised
+author: Frédéric Branchaud-Charron
+pdf: http://papers.nips.cc/paper/7263-unsupervised-transformation-learning-via-convex-relaxations.pdf
+cite:
+  authors: "Tatsunori B., Hashimoto John C. and Duchi Percy Liang"
+  title:   "Unsupervised Transformation Learning
+via Convex Relaxations"
+  venue:   "NIPS 2017"
+---
+
+The authors propose an unsupervised method to estimate the transformation matrix to describe a dataset.
+To do so, the method looks at the closest neighbor of some samples and optimize a novel convex problem.
+
+
+## Problem
+Given an input $$x \in \Re^d$$ we want a matrix $$A^{d \times d} | tAx + x = \bar x$$, where $$t$$ is a scaling factor and $$\bar x$$ is the nearest neigbhor.
```

Comment:
In the equation, what does the vertical bar mean? Generally it means "given", and is used in statistics. e.g. "what is the value of A, given that B = 42".
(Here, it seems like it means "such that", or "subject to".)

Reply:
It does mean "such that" when not used inside a probability :)